### PR TITLE
update plugin runner for directory restructure 

### PR DESCRIPTION
**Pull request by kvfairchild**
Extends #50 to work with plugin runner

### CODE REVIEW

File: brainscore_language/benchmarks/pereira2018/benchmark.py
Comment (kvfairchild) [inline brainscore_language/benchmarks/pereira2018/benchmark.py:10]:
```suggestion
```

File: brainscore_language/utils/plugins/test_plugins.py
Comment (kvfairchild) [inline brainscore_language/utils/plugins/test_plugins.py:15]:
```suggestion
		self.plugin_name = self.plugin_type + '_' + Path(self.plugin_directory).name
```

File: .travis.yml
Comment (mschrimpf) [inline .travis.yml]: I think the plugin management is such a central component that I would keep it at the top level

File: brainscore_language/metrics/linear_predictivity/test.py
Comment (mschrimpf) [inline brainscore_language/metrics/linear_predictivity/test.py:7]: this is a convention we should discuss: 
* `from .metric` imports from the local package, which I think requires an `__init__.py`. For now, I left the inits out to discourage users from importing instead of using the plugin registry
* `from metric` assumes that the working directory is this directory so would not work if called from a parent directory

File: brainscore_language/utils/plugins/test_plugins.py
Comment (mschrimpf) [inline brainscore_language/utils/plugins/test_plugins.py:8]:
```suggestion
PLUGIN_TYPES = ['benchmarks', 'data', 'metrics', 'models']
```

Comment (mschrimpf) [reply brainscore_language/utils/plugins/test_plugins.py:8]: at least once we have #51 merged

File: brainscore_language/metrics/linear_predictivity/test.py
Comment (kvfairchild) [reply brainscore_language/metrics/linear_predictivity/test.py:7]: I saw that you removed the `__init__.py`s but needed to choose between adding one here or switching out the relative import in order to get tests to pass. I chose the latter but am not attached to it - your call.

Comment (mschrimpf) [reply brainscore_language/metrics/linear_predictivity/test.py:7]: let's do this for now, but bring it up in the next check-in

**Review by mschrimpf — APPROVED** [review]

**Pull request merged**
